src/asm/aarch64/transform/inverse.rs

```diff
@@ -176,6 +247,85 @@ cpu_function_lookup_table!(
   [NEON]
 );
 
+macro_rules! impl_itx_hbd_fns {
+
+  ($TYPES:tt, $W:expr, $H:expr, [$(($OPT_LOWER:ident, $OPT_UPPER:ident)),+]) => {
+    $(
+      decl_itx_hbd_fns!($TYPES, $W, $H, $OPT_LOWER, $OPT_UPPER);
+    )*
+  };
+
+  // Loop over a list of dimensions
+  ($TYPES_VALID:tt, [$(($W:expr, $H:expr)),*], $OPT:tt) => {
+    $(
+      impl_itx_hbd_fns!($TYPES_VALID, $W, $H, $OPT);
+    )*
+  };
+
+  ($TYPES64:tt, $DIMS64:tt, $TYPES32:tt, $DIMS32:tt, $TYPES16:tt, $DIMS16:tt,
+   $TYPES84:tt, $DIMS84:tt, $OPT:tt) => {
+    // Make 2d list of tx types for each set of dimensions. Each set of
+    //   dimensions uses a superset of the previous set of tx types.
+    impl_itx_hbd_fns!([$TYPES64], $DIMS64, $OPT);
+    impl_itx_hbd_fns!([$TYPES64, $TYPES32], $DIMS32, $OPT);
+    impl_itx_hbd_fns!([$TYPES64, $TYPES32, $TYPES16], $DIMS16, $OPT);
+    impl_itx_hbd_fns!(
+      [$TYPES64, $TYPES32, $TYPES16, $TYPES84], $DIMS84, $OPT
+    );
+
+    // Pool all of the dimensions together to create a table for each cpu
+    // feature level.
+    create_wxh_hbd_tables!(
+      [$DIMS64, $DIMS32, $DIMS16, $DIMS84], $OPT
+    );
+  };
+}
+
+impl_itx_hbd_fns!(
+  // 64x
+  [(TxType::DCT_DCT, dct, dct)],
+  [(64, 64), (64, 32), (32, 64), (16, 64), (64, 16)],
+  // 32x
+  // TODO: We are excluding identity transform as tests are failing with
+  // memory misalignment for 4 cases:
+  // inv_txfm2d_add_identity_identity_16x32_neon,
+  // inv_txfm2d_add_identity_identity_32x16_neon,
+  // inv_txfm2d_add_identity_identity_32x32_neon,
+  // inv_txfm2d_add_identity_identity_8x32_neon
+  //[(TxType::IDTX, identity, identity)],
+  [],
```

Comment:
We may be able enable these transforms. The 10-bit implementations don't appear to suffer the same alignment issue.